src/build/leo/aleo-sdk/js-ts/guide.md

```diff
@@ -1 +1,1187 @@
-# Guide
+# Javascript/Typescript
+
+<!-- markdown-link-check-disable -->
+
+<a href="https://www.npmjs.com/package/@aleohq/sdk"> <img alt="Aleo SDK" src="https://img.shields.io/npm/l/%40aleohq%2Fsdk?label=NPM%20-%20Aleo%20SDK&labelColor=green&color=blue" /></a>
+
+<!-- markdown-link-check-enable -->
+
+
+## Tools for Building Zero Knowledge Web Apps
+
+The Aleo SDK is a collection of JavaScript libraries for building zero knowledge web applications in both the browser
+and node.js.
+
+## Overview
+
+Aleo provides the ability to run programs in zero knowledge. The Aleo SDK provides the tools to use these programs
+within the browser and all other levels of the web stack to build privacy preserving applications.
+
+The Aleo SDK provides the following functionality (Click to see examples):
+1. [Aleo account management](https://aleo.tools/account)
```

Comment:
https://aleo.tools seems to be down but this is an aleo issue.

Reply:
It is down for a while unfortunately. But I thought I shouldn't change it since it is very valuable resource.

Reply:
@oguzutku1745 : Can you please update the link to privable.tools please ?